mozci/push.py

```diff
@@ -1326,6 +1326,13 @@ def make_push_objects(**kwargs):
     return pushes
 
 
+def retrigger(tasks: List[TestTask], repeat_retrigger=1):
```

Comment:
Nit: one thing I didn't notice before, you should add type definitions to all arguments and a return type:
```suggestion
def retrigger(tasks: List[TestTask], repeat_retrigger: int=1) -> None:
```

otherwise mypy won't perform type checking for this function and its calls.

Reply:
Okay noted. Fixed